tasks/install_prereqs.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: ensure Auditd is installed
+  package:
+    name: auditd
+    state: present
+
+- name: Add docker gpg key
+  apt_key:
+    url: https://download.docker.com/linux/ubuntu/gpg
+    state: present
+
+- name: Add docker repo
+  apt_repository:
+    repo: "deb [arch=amd64] https://download.docker.com/linux/ubuntu xenial stable"
```

Comment:
Could you use ansible_distribution_release to build this string, and remove it from the when? I'm guessing that might make it "just work" under bionic if you do that.

Reply:
Will try it out